addons/html_builder/static/src/builder/builder_sidebar/tabs/block_tab/block_tab.xml

```diff
@@ -33,4 +40,34 @@
     </div>
 </t>
 
+<t t-name="html_builder.CustomInnerSnippet">
+    <div class="p-1 o_draggable custom-inner-content" t-att-name="snippet.title" t-att-data-category="'snippet_custom_content'" t-att-data-id="snippet.id">
+        <div class="thumbnail">
+            <div class="thumbnail-img" t-attf-style="background-image: url({{snippet.thumbnailSrc}});"/>
+            <t t-if="state.isRenaming">
+                <div class="rename-input w-100 mx-1">
+                    <input t-ref="rename-input" type="text" autocomplete="chrome-off" t-att-value="snippet.title" class="text-start" t-on-pointerdown="(ev) => ev.stopPropagation()"/>
```

Comment:
t-on-pointerdown.stop="() => {}" (Why you need to do it ?)

Reply:
Because the sidebar or he container (don't remember which one) does something on pointerDown, which means that every time I clicked on the input, this click was canceled by this something, so I could not write in it 
But I think this is temporary, when we will make the style,... of the sidebar correctly, maybe this could be removed ? I will see at that time :+1: